sunpy/net/tests/test_fido.py

```diff
@@ -489,3 +489,11 @@ def test_slice_jsoc():
 
     with pytest.warns(SunpyUserWarning) as record:
         Fido.fetch(res[0, 0:1])
+
+
+@pytest.mark.remote_data
+def test_response_block_properties():
+    res = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('aia'), a.Wavelength(171 * u.angstrom),
+                      a.vso.Sample(10 * u.minute))
+    properties = res.response_block_properties()
+    assert len(properties) == 0
```

Comment:
Why is this zero? What does the return from this function look like if there is actual data? 

I think also, since this is a test for a VSO Client method, the test should live in that file.

Reply:
output of this function is a set of properties common to all the rows,
I included this test inside Fido test because if I included it inside test_vso then I had to import Fido inside test_vso just for the one test so I thought of including the test inside the test_Fido file.

Reply:
Can't you use the vso client directly instead of fido?

Reply:
No, because this function works on unifiedResponse which is returned by FIDO.

Reply:
Is the method not implemented on the VSOClient?

Reply:
yes, I checked and it worked with vso query response. I'll change the location of the test case.